client/state/themes/selectors/get-premium-theme-price.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * External dependencies
+ */
+import i18n from 'i18n-calypso';
```

Comment:
I believe that the plan is to make the named exports go away from `i18n-calypso`, rather than the default export. The reasoning is that `i18n-calypso` is designed to be able to have multiple instances, and the named exports break this assumption. @jsnajdr will know more 🙂